README.md

```diff
@@ -213,6 +214,14 @@ The creation of new things on chains will have value. Selling them to people who
 
 * [Cryptokitties](https://www.cryptokitties.co/)
 
+## Crypto trading strategies
+
+Not unsimilar to conventional market trading, crypto markets have their own set of strategies utilized by various traders:
+1. **Algorithmic trading** - Algorithmic trading is commonly used in crypto markets to programmatically automate trading in response to varying market data. Based on defined sets of rules (including timing, price, quantity, or even a mathematical model), this type of trading not only eliminates the factor of human sentiment when making market decisions, but it also enables 24/7 automatic trading, with limited human input, enabling instant and accurate order placement and reduced risk of manual errors.
+2. **Machine Learning-influenced trading** - Machine Learning-influenced trading is another strategy becoming more popular with improvements in AI/ML. This method involves using trade execution algorithms which use Machine Learning to analyze past trends and to factor in existing market conditions in real-time. This allows for traders to make predictions based on probabilistic analysis. 
+3. **Arbitrage** - Arbitrage encompasses the simultaneous buying and selling of an item (in this case, crypto) across different markets in order to take advantage of different prices for the same asset. It is commonly exercised by crypto traders on exchanges, by purchasing crypto at a lower price than what it sells for on competitor exchanges, thus netting them a profit for equalizing the market price.
```

Comment:
looks like a good start.  i'd be curious what @ybchri thinks

Reply:
If he/she in the Telegram? I'll drop them a message and ask for feedback?

Reply:
@ybchri got any comments?

Reply:
Hey, bumping this issue thread since it's been around ~1 month. Let me know if it's ready for merge.

I'll be merging the docs site too.